File: 4-median-of-two-sorted-arrays/median-of-two-sorted-arrays.cs
public class Solution {
    public double FindMedianSortedArrays(int[] nums1, int[] nums2) {
        List<int> combo = new List<int> (nums1);
        combo.AddRange(nums2);
        combo.Sort();
        int n = combo.Count();
        if(n%2==0)
        {
            return (combo[n/2-1]+combo[n/2])/2.0f;
        }
        else
        {
            return combo[n/2];
        }
    }
}

File: 4-median-of-two-sorted-arrays/median-of-two-sorted-arrays.py
class Solution:
    def findMedianSortedArrays(self, nums1: List[int], nums2: List[int]) -> float:
        nums=sorted(nums1+nums2)
        n=len(nums)
        if n%2==0:
            mid1, mid2 = nums[n//2-1], nums[n//2]
            median=(mid1+mid2)/2
        else:
            median=nums[n//2]
        return median